server/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Caution when editing on windows the EOF should be LF
+
+psql -v ON_ERROR_STOP=1 --host "postgres" --username "dofuslab" --dbname "dofuslab" <<-EOSQL
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
```

Comment:
This is just included on most newer versions of postgres I think? At least, I don't think I've had to install it manually using 12.3.

Reply:
tried it without this and it gives an error that its missing, but i really didn't try fiddling with images maybe there is some image that has this included